src/app/tests/suites/TestCluster.yaml

```diff
@@ -707,6 +707,20 @@ tests:
       response:
           value: 0
 
+    # Tests for Nullable String attribute
```

Comment:
Put this in the existing section with tests for nullable_char_string?  That would help notice why the test is failing with this change.